local/rebac-admin/README.md

```diff
@@ -0,0 +1,52 @@
+# QA ReBAC Admin Backend
+
+To QA locally the ReBAC Admin Backend you need to:
+
+## Change the rebac authentication middleware at `internal/middleware/authn.go` to:
```

Comment:
This feels a bit flimsy because the code below will not stay up to date as the actual code changes.
Also, we don't mention what the change is doing?
Could we not run the tests with the default admin user in the docker compose?

Reply:
this change is the easiest way to remove the authentication but keep the user in the context for the rebac-admin stuff to work.
Otherwise we need to create a cookie and use it in the bash script, or something like taht

Reply:
I think the best way to QA this is honestly a set of Cryprus tests running against a local UI. Not like this... ANd the tests are the responsibility of the webteam.